inlong-dataproxy/dataproxy-docker/dataproxy-docker.sh

```diff
@@ -21,6 +21,9 @@ local_ip=$(ifconfig | grep inet | grep -v inet6 | grep -v "127.0.0.1" | awk '{pr
 # config
 cd "${file_path}/"
 common_conf_file=./conf/common.properties
+pulsar_conf_file=./conf/dataproxy-pulsar.conf
+tube_conf_file=./conf/dataproxy-tube.conf
```

Comment:
here we can add CONFIG_FILE="dataproxy-${MQ_TYPE}.conf" to define different configuration file.

Reply:
done